src/x_net.c

```diff
@@ -65,12 +65,12 @@ const char* INET_NTOP(int af, const void *src, char *dst, socklen_t size) {
     }
     else
         return NULL;
-    if (WSAAddressToString((struct sockaddr *)&addr, addrlen, 0, dst,
```

Comment:
Why the "A" suffix?

Reply:
Because many Windows API functions are actually macros which expand to either the short (*A) or the wide (*W) version, depending on whether the code is compiled with `-DUNICODE`. In such a case, the whole code must be "unicode" aware (e.g. always using TCHAR instead of char, _T("foo") for string literals, etc.)

Otherwise it's good practice to explicitly use the matching version, which in our case is the A version because of `char *`. There are actually quite some places in Pd which don't use wide characters for Windows (and implicitly use the short version of the API function, e.g. `LoadLibrary` in `s_loader.c`) but cleaning this up is another project :-)